ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/fragmentPage/v1/fragmentPage/footerlibs.html

```diff
@@ -0,0 +1,28 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Even this looks a copy paste, please extend from existing forms page core component, and remove these files. It would inherit the same

Reply:
Yes, this will be removed as this was for static implementation.